.babelrc

```diff
@@ -3,7 +3,9 @@
     [
       "env",
       {
-        "node": "8.10"
+        "targets": {
```

Comment:
What for?

Reply:
say to babel to  compile against the node version 8.10

Reply:
@Vadorequest see up